### PR TITLE
[ADD] personal_trainer: Personal Trainer

### PR DESCRIPTION
**Pull request by dhrs-odoo**
This industry package includes everything you need to run your coaching business
efficiently with Odoo.

Discover everything you need to know about this package by visiting the Knowledge
App after installing it.

Basics
Use the Appointment app to schedule and manage client sessions effortlessly.
Leverage the CRM to track leads and nurture client relationships.
Utilize the Sales app to create and manage your coaching services and packages.
Employ the Project app for long-term client program management and progress
tracking.
Use the Invoicing app to bill clients and manage your finances.

Included customizations
This industry relies only on standard Odoo features and does not include any
customization.

Task-4160636

### CODE REVIEW

File: sports_coach/demo/project_project.xml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo noupdate="1">
+    <record id="project_project_1" model="project.project">
+        <field name="name">Internal</field>
+        <field name="account_id" ref="account_analytic_account_1"/>
+    </record>
+    <record id="project_project_2" model="project.project">
+        <field name="account_id" ref="account_analytic_account_2"/>
```
Comment (vava-odoo) [inline sports_coach/demo/project_project.xml:8]: I'm pretty sure we don't have to define it ourself, it is automated

File: sports_coach/demo/project_project.xml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+        <field name="tag_ids" eval="[Command.set([ref('project_tags_1')])]"/>
```
Comment (vava-odoo) [inline sports_coach/demo/project_project.xml:9]: The tag can moved to data

File: sports_coach/demo/res_partner.xml [hunk 3]
```diff
@@ -0,0 +1,73 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo noupdate="1">
+    <record id="res_partner_13" model="res.partner">
+        <field name="name">John Smith</field>
+        <field name="image_1920" type="base64" file="sports_coach/static/src/binary/res_partner/13-image_1920"/>
+        <field name="user_id" ref="base.user_admin"/>
+        <field name="email">john.smith@email.com</field>
+        <field name="partner_share" eval="True"/>
```
Comment (vava-odoo) [inline sports_coach/demo/res_partner.xml:8]: Looks like a computed store field. Maybe add a test to remove those
```suggestion
```

**Review by vava-odoo — CHANGES_REQUESTED** [review]
Almost there, just a few things.
We also need to also add back the timesheet (see example in gardening or odoo_partner).
Thanks @dhrs-odoo

File: sports_coach/data/crm_stage.xml [hunk 4]
```diff
@@ -0,0 +1,24 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo noupdate="1">
+<record id="crm.stage_lead1" model="crm.stage">
```
[the remark this answers is not in the document]
Comment (dhrs-odoo) [reply sports_coach/data/crm_stage.xml:3]: Yes as the stages are different from the standard crm stages so we need to edit the stages

**Comment by dhrs-odoo** [comment]
Hello @vava-odoo 
I have incorporated the changes 
Please have a look at it once you are free
Thank You :innocent:

File: sports_coach/__manifest__.py [hunk 5]
```diff
@@ -0,0 +1,51 @@
+{
+    'name': 'Sports coach',
+    'version': '1.0',
+    'category': 'Health and Fitness',
+    'description': "",
+    'depends': [
+        'accountant',
```
Comment (vava-odoo) [inline sports_coach/__manifest__.py:7]: Do we need accountant?! I think you can activate analytic account without it

**Review by vava-odoo — CHANGES_REQUESTED** [review]
Thanks, but didn't you forget the timesheets?

**Comment by vava-odoo** [comment]
Also the tag should be removed for automatically generated projects

**Comment by dhrs-odoo** [comment]
Hello @vava-odoo Changes has been implemented
Thank You :innocent:

File: personal_trainer/static/description/index.html [hunk 6]
```diff
@@ -0,0 +1,26 @@
+<h2><strong>Odoo for Sports Coaches</strong></h2>
```
Comment (vava-odoo) [inline personal_trainer/static/description/index.html:1]: :eyes:

File: personal_trainer/demo/account_analytic_line.xml [hunk 7]
```diff
@@ -0,0 +1,47 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo>
+    <record id="account_analytic_line_1" model="account.analytic.line">
+        <field name="employee_id" ref="hr.employee_admin" />
+        <field name="timesheet_invoice_type">billable_fixed</field>
+        <field name="name">/</field>
+        <field name="date">2024-09-16</field>
+        <field name="project_id" ref="project_project_3"/>
```
Comment (vava-odoo) [inline personal_trainer/demo/account_analytic_line.xml:8]: wrong project, you may want to link to the one created from the sale order

File: personal_trainer/demo/account_analytic_line.xml [hunk 8]
```diff
@@ -0,0 +1,47 @@
...
+        <field name="task_id" ref="project_task_18"/>
```
Comment (vava-odoo) [inline personal_trainer/demo/account_analytic_line.xml:9]: This sale order line generates a project, not a task. Not sure it works as expected

File: personal_trainer/demo/account_analytic_line.xml [hunk 9]
```diff
@@ -0,0 +1,47 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo>
+    <record id="account_analytic_line_1" model="account.analytic.line">
+        <field name="employee_id" ref="hr.employee_admin" />
+        <field name="timesheet_invoice_type">billable_fixed</field>
+        <field name="name">/</field>
+        <field name="date">2024-09-16</field>
```
Comment (vava-odoo) [inline personal_trainer/demo/account_analytic_line.xml:7]: Need something relative to the installation date
```suggestion
        <field name="date" eval="(datetime.today() - timedelta(days=1)).strftime('%Y-%m-%d')"/>
```

**Review by vava-odoo — CHANGES_REQUESTED** [review]

**Comment by dhrs-odoo** [comment]
Hello @vava-odoo 
Done with the changes
Thank You :innocent:

File: industry_real_estate/data/ir_ui_views.xml [hunk 10]
```diff
@@ -82,7 +82,7 @@
         <field name="arch" type="xml">
             <kanban>
                 <templates>
-                    <t t-name="kanban-box">
+                    <t t-name="card">
```
Comment (vava-odoo) [inline industry_real_estate/data/ir_ui_views.xml:85]: Nice, but could you do this in another PR so that it is quickly fixed?

File: personal_trainer/demo/account_analytic_line.xml [hunk 11]
```diff
@@ -0,0 +1,47 @@
...
+        <field name="unit_amount">2.0</field>
+        <field name="department_id" ref="hr.dep_administration" />
+    </record>
+    <record id="account_analytic_line_2" model="account.analytic.line">
+        <field name="employee_id" ref="hr.employee_admin" />
+        <field name="timesheet_invoice_type">billable_fixed</field>
+        <field name="name">/</field>
+        <field name="date" eval="(datetime.today() - timedelta(days=1)).strftime('%Y-%m-%d')"/>
+        <field name="project_id" ref="project_project_3"/>
+        <field name="task_id" search="[('sale_line_id', '=', ref('sale_order_line_1'))]"/>
```
Comment (vava-odoo) [inline personal_trainer/demo/account_analytic_line.xml:20]: This doesn't work. Maybe remove the project_id?

File: personal_trainer/demo/account_analytic_line.xml [hunk 12]
```diff
@@ -0,0 +1,47 @@
...
+        <field name="unit_amount">1.0</field>
+        <field name="department_id" ref="hr.dep_administration" />
+    </record>
+    <record id="account_analytic_line_3" model="account.analytic.line">
+        <field name="employee_id" ref="hr.employee_admin" />
+        <field name="timesheet_invoice_type">billable_fixed</field>
+        <field name="project_id" ref="project_project_2"/>
+        <field name="task_id" search="[('sale_line_id', '=', ref('sale_order_line_4'))]"/>
+        <field name="name">/</field>
+        <field name="date" eval="(datetime.today() - timedelta(days=1)).strftime('%Y-%m-%d')"/>
```
Comment (vava-odoo) [inline personal_trainer/demo/account_analytic_line.xml:31]: try to have different dates

File: personal_trainer/demo/account_analytic_line.xml (hunk 11 above)
Comment (dhrs-odoo) [reply personal_trainer/demo/account_analytic_line.xml:20]: It works I have checked this

Comment (vava-odoo) [reply personal_trainer/demo/account_analytic_line.xml:20]: No, because it is counted in the template project, which doesn't make sense
image

Comment (dhrs-odoo) [reply personal_trainer/demo/account_analytic_line.xml:20]: image

**Comment by dhrs-odoo** [comment]
Hello @vava-odoo 
Implemented the changes
Thank You :innocent:

**Review by vava-odoo — APPROVED** [review]
Good! Before the merge, could you:
- update the pot;
- add the entry in the .tx/config;
- merge last commit in the first one?

Thanks!

**Comment by dhrs-odoo** [comment]
Hello @vava-odoo 
I have applied your comments
Thank You :innocent:

**Comment by vava-odoo** [comment]
@robodoo r+ rebase-ff
